### ISSUE TITLE
Keybase integration

### ISSUE THREAD

**Issue by yshalenyk** [justwatchcom/gopass#735]
Is there way to use keybase as a backend in gopass?

**Comment by yshalenyk** [comment]
@jicksta agreed, it works. but still, its a bit hacky way :) Also it would be awesome feature to use key from keybase or even direct keybase encrypt/decrypt.

**Comment by jicksta** [comment]
Personally I'd prefer the encrypted git remote option over some bespoke protocol.

The thing I think I lack (that I haven't bothered figuring out yet) is eager or lazy bunzipped backups of `~/.gnupg` and `~/.password-store` to a private KBFS subdir.
